packages/ui/src/utils/nav.utils.js

```diff
@@ -0,0 +1,39 @@
+import { is } from 'ramda';
```

Comment:
Ramda is a foundation library so that we don't need to care too much about the tree shaking stuff. I think it's better to have an "R" namespace as it's introduced in the [official README](https://github.com/ramda/ramda#installation). However, for most of the other libraries, I agree that your import statement is more friendly for tree shaking if we combining use with some babel plugin like [babel-plugin-import](https://www.npmjs.com/package/babel-plugin-import).

```javascript
import * as R from 'ramda';
```